networkx/drawing/layout.py

```diff
@@ -617,3 +619,178 @@ def flatten(l):
         else:
             yield el
 
+
+def kamada_kawai(G, pos=None, K=1.0, L=0.1, iterations=None,
+                 tolerance=1, D=0.0, weight='weight'):
+    """Position nodes using Kamada-Kawai force-directed algorithm.
+
+    Parameters
+    ----------
+    G : NetworkX graph
+
+    pos : dict or None  optional (default=None)
+       Initial positions for nodes as a dictionary with node as keys
+       and values as a list or tuple.  If None, then use random initial
+       positions.
+
+    K : float (default=1.0)
+       A constant which determines the strength of the spring.
+
+    L : float (default=0.1)
+        L a constant which determines desirable
+        length of a single edge in the display plane.
+
+    iterations : int  optional (default=None)
+       Number of maximum iterations of the algorithm.
+       If None, then 10 * n iterations will be completed.
```

Comment:
Wherever you see "None" in the documentation, it should be ``` ``None`` ```. Similarly, you should have ``` ``10 * n`` ``` wherever you have code.